Dockerfile.alpine-gocd-server

```diff
@@ -0,0 +1,41 @@
+# Build using: GO_VERSION=16.x.x-xxxx docker build -f Dockerfile.alpine-gocd-server --tag=alpine-gocd-server .
+
+FROM delitescere/jdk:latest
+#ENV GO_VERSION=16.5.0-3305
+ENV GO_VERSION=16.6.0-3590
+MAINTAINER GoCD <go-cd-dev@googlegroups.com>
+
+# GoCD scripts used here work with ash -- bash not needed.  Only git support is bundled.
+RUN apk --no-cache add git
```

Comment:
> If you're in agreement I can add some documentation (where?) describing how to use it as a base image.

Sounds good, I was under the assumption they'd be a few megs. I think adding a note somewhere in the README is a good start.